tests/integ/test_inference_recommender.py

```diff
@@ -206,7 +206,7 @@ def default_right_sized_unregistered_model(sagemaker_session, cpu_instance_type)
                 ir_job_name,
             )
         except Exception:
-            sagemaker_session.delete_model(ModelName=sklearn_model.name)
+            sagemaker_session.delete_model(model_name=sklearn_model.name)
```

Comment:
[`session.py`](https://github.com/aws/sagemaker-python-sdk/blob/d960e49a245d3fbccf5315ad646fc149126f3390/src/sagemaker/session.py#L4213) use model_name param not ModelName

Reply:
Found in personal stack error like:

```
except Exception:
	>               sagemaker_session.delete_model(ModelName=sklearn_model.name)
	E               TypeError: Session.delete_model() got an unexpected keyword argument 'ModelName'
tests/integ/test_inference_recommender.py:209: TypeError
```